myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java

```diff
@@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
       //something is broken
       throw new RuntimeException("Could not find valid updateFile Method");
     }
+    
+    startInternal();
```

Comment:
HI Darin, I added this because the Junit test I wrote was throwing a NPE, this fixed it, and then I forgot to go back and figure out a more permanent solution that does not involve multiple invocations of startInternal.  I'll do that ASAP and check back in later today